tests/gzip/gzip-env.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+# Test the obsolescent GZIP environment variable.
```

Comment:
I don't remember exactly, but it seems this variable resulted in security troubles for `gzip` (the variable could be abused to make `gzip` do unwanted silent actions).
The consequence of which was to either disable or limit the scope of this variable.
Could be worth investigating, to know which level of support is actually desirable.